### PR TITLE
Update SectorVocabulary.xml

### PR DESCRIPTION
**Pull request by andreaszenasidi**
Changes made to sector vocabulary XML as requested in:
https://github.com/IATI/IATI-Codelists-NonEmbedded/issues/336
https://github.com/IATI/IATI-Codelists-NonEmbedded/issues/338

### CODE REVIEW

File: xml/SectorVocabulary.xml [hunk 1]
```diff
@@ -89,7 +89,7 @@
                 <narrative>A value from the top-level list of UN sustainable development goals (SDGs) (e.g. ‘1’)</narrative>
                 <narrative xml:lang="fr">Valeur tirée de la liste des principaux objectifs de développement durable (ODD) de l'ONU (p. ex. '1')</narrative>
             </description>
-            <url>https://sustainabledevelopment.un.org/?menu=1300</url>
+            <url>http://reference.iatistandard.org/codelists/unsdg-goals/</url>
```
Comment (nosvalds) [inline xml/SectorVocabulary.xml:92]:
```suggestion
            <url>http://reference.iatistandard.org/codelists/unsdg-goals</url>
```

Comment (nosvalds) [reply xml/SectorVocabulary.xml:92]: Small suggestion, leave off `/` if it's the only thing at the end.

**Review by nosvalds — APPROVED** [review]
Looks good to me. One small comment/update.

File: xml/SectorVocabulary.xml (hunk 1 above)
Comment (nosvalds) [inline xml/SectorVocabulary.xml]:
```suggestion
            <url>http://reference.iatistandard.org/codelists/unsdg-goals</url>
```

File: xml/SectorVocabulary.xml [hunk 2]
```diff
@@ -101,7 +101,7 @@
                 <narrative>A value from the second-level list of UN sustainable development goals (SDGs) (e.g. ‘1.1’)</narrative>
                 <narrative xml:lang="fr">Valeur tirée de la liste des sous-objectifs de développement durable (ODD) de l'ONU (p. ex. '1.1')</narrative>
             </description>
-            <url>http://unstats.un.org/sdgs/indicators/indicators-list/</url>
+            <url>http://reference.iatistandard.org/codelists/unsdg-targets/</url>
```
Comment (nosvalds) [inline xml/SectorVocabulary.xml:104]:
```suggestion
            <url>http://reference.iatistandard.org/codelists/unsdg-targets</url>
```

**Review by nosvalds — APPROVED** [review]

**Review by amy-silcock — APPROVED** [review]
These look good to me!

**Comment by nosvalds** [comment]
I'm merging this now. I'm not going to go through the whole process to generate the Website pages and push them there right now as it's quite a bit of overhead. These will be included the next time we do an update of the website SSOT pages.

**Comment by andylolz** [comment]
> I'm not going to go through the whole process to generate the Website pages and push them there right now as it's quite a bit of overhead. These will be included the next time we do an update of the website SSOT pages.

It would be cool to auto-deploy via a github action.

It was discussed for the old reference site in IATI/IATI-Standard-SSOT#164 and IATI/IATI-Standard-SSOT#297. In fact it was implemented here: https://referencebot.github.io

**Pull request merged**